library/Icinga/Web/Dashboard/Common/ModuleDashlet.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/* Icinga Web 2 | (c) 2022 Icinga GmbH | GPLv2+ */
+
+namespace Icinga\Web\Dashboard\Common;
+
+trait ModuleDashlet
```

Comment:
Is this supposed to be used multiple times? Currently there is a single usage. If it keeps that way, I don't see why these methods should be in a trait.